python/lsst/ts/adamSensors/mockModel.py

```diff
@@ -0,0 +1,89 @@
+from pymodbus.client.sync import ModbusTcpClient as ModbusClient
+from pymodbus.exceptions import ConnectionException, ModbusIOException
+
+
+class AdamModel:
```

Comment:
Uhm... why do you have a `AdamModel` in `mockModel.py` (which should actually be `mock_model.py`) and another one in `model.py`. You can probably merge both into one.